packages/flutter/test/material/app_test.dart

```diff
@@ -249,7 +249,7 @@ void main() {
   });
 
   testWidgets('Return value from pop is correct', (WidgetTester tester) async {
-    late Future<Object> result;
+    late Future<Object?> result;
```

Comment:
The internal test failure is similar to what caused this test to fail

Reply:
It's slightly different, fix is in cl/338282385.